test/unit/model/article-information.test.ts

```diff
@@ -44,12 +47,53 @@ describe('ArticleInformation', () => {
         "publicationDate": "",
         "publisherId": "",
         "subjects": [],
-        "volume": "",
+        "volume": ""
       });
+
+      expect(articleInfo.licenseText!.doc.textContent).toBe('');
     });
 
     it('Creates an ArticleInformation from JSON', () => {
-      const articleInfo = new ArticleInformation({});
+      const articleInfo = new ArticleInformation({
+        articleDOI: "10.7554/eLife.00104",
+        articleType: "Insight",
+        copyrightStatement: "",
+        dtd: "1.2",
+        elocationId: "e00104",
+        licenseText: {
+          "doc": {
+            "content": [{"type": "paragraph"}],
+            "type": "doc",
+          },
+          "selection": {
+            "anchor": 0,
+            "head": 0,
+            "type": "text",
+          }
+        },
+        licenseType: "CC-BY-4",
+        publicationDate: "2019-11-30",
+        publisherId: "00104",
+        subjects: ['Cell Biology', 'Genetics and Genomics'],
+        volume: "8",
+      });
+
+      expect(articleInfo).toStrictEqual(expect.objectContaining({
+        _id: 'unique_id',
+        articleDOI: "10.7554/eLife.00104",
+        articleType: "Insight",
+        copyrightStatement: "",
+        dtd: "1.2",
+        elocationId: "e00104",
+        licenseText: expect.any(EditorState),
+        licenseType: "CC-BY-4",
+        publicationDate: "2019-11-30",
+        publisherId: "00104",
+        subjects: ['Cell Biology', 'Genetics and Genomics'],
+        volume: "8"
+      }));
+
+      expect(articleInfo.licenseText!.doc.textContent).toBe('');
```

Comment:
Should this have a value present to ensure we're configuring the `EditorState.fromJSON` correctly with a the value `licenseText` value?

Reply:
Maybe `Three miles long EULA` for consistency between the xml test and this one?